src/device_trezor/trezor/update_protob.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
The script only seems to use POSIX shell compatible features, so you might consider just `sh` vs `bash` here to make the program available on more systems.

Reply:
Arrays are not supported in POSIX sh. I changed to script to make it compatible.

Thanks for reviewing. :)

Reply:
You're right, I forgot POSIX still doesn't support arrays, but the updated version without arrays looks good.